beacon_node/eth1/src/deposit_cache.rs

```diff
@@ -76,10 +143,13 @@ impl SszDepositCache {
 /// Mirrors the merkle tree of deposits in the eth1 deposit contract.
 ///
 /// Provides `Deposit` objects with merkle proofs included.
+#[cfg_attr(test, derive(PartialEq))]
 pub struct DepositCache {
     logs: Vec<DepositLog>,
     leaves: Vec<Hash256>,
     deposit_contract_deploy_block: u64,
+    finalized_deposit_count: Option<u64>,
+    finalized_block_height: Option<u64>,
```

Comment:
It seems we almost always `.unwrap_or(0)` these fields, thoughts on making them `u64` instead with 0 as the default? Do we ever need to distinguish between the `None` and `Some(0)` cases?

Reply:
good catch! I've refactored the code to just make it a `u64` now :)